woocommerce/woocommerce-ios
# Quick Order: Rename "Quick Pay" to "Quick Order"

# Why

As instructed on p91TBi-6iQ#comment-6850 this PR renames all occurrences of "Quick Pay" in favor of "Quick Order"

# Testing Steps

- Just make sure the app compiles, runs and unit tests pass!

Update release notes:
